app/views/user_mailer/password_reset.html.erb

```diff
@@ -1,5 +1,7 @@
-<h1>User#password_reset</h1>
+<h1>Password reset</h1>
 
 <p>
-  <%= @greeting %>, find me in app/views/user_mailer/password_reset.html.erb
+  Hi, <%= @user.name%>,
+  Please click the link below to reset your password. It will expire in 15 minutes.
```

Comment:
Add something like "if you did not request for a password reset, please disregard this email."